src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/TaskConfigurationService.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.wataskmonitor.services;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.wataskmonitor.clients.TaskConfigurationClient;
+import uk.gov.hmcts.reform.wataskmonitor.models.CamundaTask;
+
+import java.util.List;
+
+@Component
+@Slf4j
+public class TaskConfigurationService {
+
+    private final TaskConfigurationClient taskConfigurationClient;
+
+    @Autowired
+    public TaskConfigurationService(TaskConfigurationClient taskConfigurationClient) {
+        this.taskConfigurationClient = taskConfigurationClient;
+    }
+
+    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+    public void configureTasks(List<CamundaTask> camundaTasks, String serviceToken) {
+        camundaTasks.forEach(task -> {
+            log.info("Configuring tasks...");
```

Comment:
log.info("Configuring tasks..."); -> should it be "Configuring task" as its per task. I got here by looking at the test coverage which is great.

Reply:
I moved log above and outside the loop